### DISCUSSION TITLE
why leaderboard is so different

### DISCUSSION THREAD

**Discussion by Jacob4565** [ppy/osu#19177] [Q&A] [answered]
so now i played osu lazer 2022.716.1 and when im play few maps i realized thel eaderboard has change  like this
![Screenshot (162)](https://user-images.githubusercontent.com/108603366/179384467-ec7b8e18-3bae-4fb3-b8e9-3fb4e424581f.png)
its change to much, what i mean the order leaderboard has change, at first the leaderboard have famous osu player like lifeline, mreek, whitecat etc but is change i dont know wtf is player

sorry for my english bad

**Comment by frenzibyte** [comment, marked as answer]
What you're seeing here is a lazer leaderboard, you can submit scores and see them right there.

The classic/stable leaderboard will eventually return.

**Reply by nagi-desuuu** [reply]
Refer to #19126 for full context.
